scripts/build.js

```diff
@@ -1,17 +0,0 @@
-var spawn = require('cross-spawn');
-var Habitat = require('habitat');
-
-Habitat.load();
-
-var env = new Habitat();
-var buildCommand = 'start:dev';
-
-if (env.get('NPM_CONFIG_PRODUCTION')) {
-  buildCommand = 'start:prod';
-}
-
-var npm = spawn('npm', ['run', buildCommand], { stdio: 'inherit' });
-
-npm.on('close', function(code) {
-  process.exit(code);
-});
```

Comment:
We now run the repo like advocacy, which ditched this method of prod/dev. A refresher is this `npmm start` is simply a dev only thing, that does what prod does + watch.